packages/core/src/sync-store/store.ts

```diff
@@ -181,6 +181,7 @@ export interface SyncStore {
         log: Log;
         block: Block;
         transaction: Transaction;
+        checkpoint?: Checkpoint;
```

Comment:
When would checkpoint be undefined?

Reply:
the log's tables checkpoint column is currently nullable. but i think once it gets to this part of the code its impossible to be undefined so i can change it

Reply:
fixed :)